doc/content/source/materials/NEMLStress.md

```diff
@@ -8,20 +8,32 @@ This material model provides an interface to the constitutive models for structu
 
 NEML permits the definition of complex material constitutive models, the details of which are defined in an XML file that is independent of the MOOSE input file. The required inputs for this model on the MOOSE side simply provide the location of the XML file, which contains a database of potentially multiple material models, and the name of the model within that database to be used.  This model computes the stress, elasticity tensor, and material Jacobian.
 
-Using this model requires that the NEML source code is present, and that the prerequisite libraries are available. NEML is provided as a submodule in BlackBear under the directory `blackbear/contrib/neml`. The up-to-date version of the NEML submodule is obtained using:
+Using this model requires that the NEML source code is present, and that the prerequisite libraries are available. 
+NEML is provided as a submodule in BlackBear under the directory `blackbear/contrib/neml`. 
+The up-to-date version of the NEML submodule is obtained using:
+
 ```
 git submodule init
 git submodule update
 ```
-in the `blackbear` directory.
 
-NEML is compiled using the MOOSE build system, so no separate compilation step is required to build the library. However, NEML has dependencies on the BOOST library that is not present in the default MOOSE environment.
+in the `blackbear` directory.
+NEML is compiled using the MOOSE build system, so no separate compilation step is required to build the library. 
+However, NEML has dependencies on the BOOST library that are not present in the default MOOSE environment.
+To build with boost, one must simply load the `boost` module provided with MOOSE:
 
-To build with BOOST, one must simply load the `boost` module provided with MOOSE:
 ```
 module load advanced_modules boost
 ```
-and add the `--with-boost` option to the `update_and_rebuild_libmesh.sh` script when compiling libMesh to instruct it to use the full BOOST instalation provided by the `boost` module, rather than the subset of BOOST that is distributed with libmesh.
+
+and rebuild libMesh with the `--with-boost` option:
+
+```
+./moose/update_and_rebuild_libmesh.sh --with-boost
```

Comment:
I was just about to merge this, and then I realized that the path here is wrong. From the blackbear directory, it's `./moose/scripts/update_and_rebuild_libmesh.sh`

Reply:
can you run `update_and_rebuild_libmesh.sh` outside the moose directory?

Reply:
Actually, I'm not sure.  I always run it from the moose directory.  Maybe better to say:
```
cd moose
./scripts/update_and_rebuild_libmesh.sh --with-boost
```